test/test_helper.rb

```diff
@@ -1,6 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'daimon_skycrawlers'
 
+require 'faraday'
```

Comment:
This will be resolved by https://github.com/bm-sms/daimon-skycrawlers/pull/18 to stop using `autoload` :rocket: